spec/smartcar/helpers/auth_helper.rb

```diff
@@ -42,10 +42,13 @@ def run_auth_flow(authorization_url)
 
       wait = Selenium::WebDriver::Wait.new(:timeout => 10)
 
-      wait.until {
+      if(test_email.nil?)
```

Comment:
If we give one of our test email IDs It doesn't seem to show the approve button.